unicorn_binance_websocket_api/unicorn_binance_websocket_api_socket.py

```diff
@@ -126,6 +126,7 @@ async def start_socket(self):
                                 received_stream_data = self.unicorn_fy.binance_com_futures_websocket(received_stream_data_json)
                             elif self.exchange == "binance.com-futures-testnet":
                                 received_stream_data = self.unicorn_fy.binance_com_futures_websocket(received_stream_data_json)
+                            # TODO coin margined futures (What to do about it?)
```

Comment:
No sure what to do here

Reply:
We need to update unicorn-fy too. Since i dont know whats returned from the endpoint we can start with a "try" and map coin-futures in unicorn-fy to std websocket convertation. If you want you can try, if not i can do it too, its not much work.

Reply:
we will call it "binance_com_coin_futures_websocket()`

Reply:
I'll try. I wanted to look into the other packages anyways.

Reply:
Thanks! https://github.com/oliver-zehentleitner/unicorn-fy/issues/26

Reply:
How do you want this line to be fixed now? Update the comment?

Reply:
just add it with a call to `binance_com_coin_futures_websocket()`, but maybe we should discuss https://github.com/oliver-zehentleitner/unicorn-binance-websocket-api/pull/163#issuecomment-819378937 first

Reply:
Ah ok you accept the broken call to non existing function because you'll not release it yet. I replied to https://github.com/oliver-zehentleitner/unicorn-binance-websocket-api/pull/163#issuecomment-819378937 in https://github.com/oliver-zehentleitner/unicorn-binance-websocket-api/pull/163#issuecomment-819381908

Reply:
yes and the unicorn-fy thing will be done very soon (from you or from me...) so i dont worry about it.

Reply:
@oliver-zehentleitner done